srcpkgs/upower/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'upower'
 pkgname=upower
-version=1.90.0
+version=1.90.1
```

Comment:
.2 is out

Reply:
.2 depends on libudev > 251, void ships that using eudev and it's currently at 243. (https://github.com/void-linux/void-packages/blob/1556a665c716e61dc99f0adacf0e4fbb607a6903/srcpkgs/eudev/template#L3)